source/_integrations/nest.markdown

```diff
@@ -49,7 +49,7 @@ You are in control of the information and capabilities exposed to Home Assistant
 
 <div class='note'>
 
-The Nest Smart Device Management (SDM) API **requires a US$5 fee**.
+The Nest Smart Device Management (SDM) API **requires a US$5 fee**. BEFORE PAYING, make sure your device is [supported](https://developers.google.com/nest/device-access/supported-devices). Speakers are NOT included.
```

Comment:
Full capitalized words are generally considered to be screaming. Let's not do that, as there is no reason for it either. Additionally, suggesting to make it similar to the change made above:

```suggestion
The Nest Smart Device Management (SDM) API **requires a US$5 fee**. Before buying, make sure your device is [supported](https://developers.google.com/nest/device-access/supported-devices).
```

Reply:
Hi @frenck, I've made an additional commit, using italics in lieu of capitals. (Although, if capitals = screaming, perhaps that's not a bad thing in this case).

Reply:
Screaming is always bad. There if not much to discuss in that regard.